index.html

```diff
@@ -32,6 +32,9 @@ <h2>Filter Starred Ideas</h2>
         <textarea class="comment-text" rows=4 type="text" name="comment-text" placeholder="Leave a Comment!"></textarea>
         <button type="submit" class="submit-comment" disabled>Add Comment</button>
       </div>
+      <h1 class="comment-header">Comments</h1>
+      <div class="comment-display">
```

Comment:
I like the way we integrated the comments to be displayed in the same 'modal' window as the comment submission form. So neat!